recipes/docker.rb

```diff
@@ -216,6 +216,28 @@
             })
 end
 
+cookbook_file node['hops']['docker']['hopsfsmount-seccomp-profile'] do
+  source 'hopsfsmount_seccomp_profile.json'
+  owner 'root'
+  mode '0755'
+  action :create
+end
+
+hopfsmount_apparmor_profile="/etc/apparmor.d/#{node['hops']['docker']['hopfsmount-apparmor-profile']}"
+cookbook_file hopfsmount_apparmor_profile do
+  source 'hopsfsmount_apparmor_profile'
+  owner 'root'
+  mode '0755'
+  action :create
+end
+
+bash 'move-docker-images' do
+  user 'root'
+  code <<-EOH
+      apparmor_parser -r -W #{hopfsmount_apparmor_profile}
+  EOH
```

Comment:
Same as above, add a guard to execute this only when contents manager is hopsfs-mount